### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dbr0**
A bit more straightforward usage example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,22 @@ You can now configure Googlmapper in a few simple steps. Open `app/config/packag
 
 ## Usage
 
-It's really as simple as using the Mapper class in any Controller / Model / File you see fit with:
+Initialize the map in your controller `MyController.php` as:
```
Comment (bradcornford) [inline README.md]: Can you keep the sentence above, then add the code example under a ###Example title section.

**Comment by bradcornford** [comment]
This looks good to me. I'm just working on a fix for an issue thats been spotted, then i'll progress this into master.

File: README.md (hunk 1 above)
Comment (dbr0) [inline README.md]: I've created another pull request as I don't know how to edit this one.

**Comment by dbr0** [comment]
Np, trying to contribute (even if it's something as simple as this) and figure out this push-pull-fork thing.

**Pull request closed without merging**